go.mod

```
@@ -1,6 +1,6 @@
 module helm.sh/helm/v3
 
-go 1.24
+go 1.24.0
```

```suggestion
go 1.24
```
Was the `0` patch required? My understanding is this would force projects that are dependent on helm sdk to use 1.24.0 rather than a later 1.24 patch.

---

Yes sadly. I was not able to have a green test if I remove the patch. I don't know if there is other way. I would prefer to stick to `1.24`